test/mCODECancerRelatedRadiationProcedure.test.js

```diff
@@ -1,9 +1,7 @@
 const path = require('path');
-// eslint-disable-next-line object-curly-newline
-const { defaultLoadElm, loadJSONFixture, defaultLoadValuesets, mapValueSets } = require('cql-testing-harness');
-const { setup } = require('./setup');
+const { execute, defaultLoadElm, loadJSONFixture, defaultLoadValuesets, mapValueSets } = require('cql-testing-harness');
 
-let testSetup;
+let executionResults;
 beforeAll(() => {
   // Set up necessary data for cql-execution
   const valueSets = defaultLoadValuesets(defaultLoadValuesets);
```

Comment:
DP1 - passing the function as an argument. Would you mind fixing? 
```suggestion
  const valueSets = defaultLoadValuesets();
```